src/main/java/org/apache/commons/daemon/DaemonPermission.java

```diff
@@ -181,11 +186,10 @@
      *
      * @param target The target name of this permission.
      * @throws IllegalArgumentException If the specified target name is not
-     *                supported.
+     *                                  supported.
```

Comment:
No extra indent please. You can make the line length for code and Javadoc 120 characters to avoid this kind of wrapping. So do not change it or, if you do, use 120 line lengths.